codalab/lib/completers.py

```diff
@@ -213,6 +213,18 @@ def __call__(self, prefix, action=None, parser=None, parsed_args=None):
             warn('Error: {}'.format(ex))
 
 
+class DockerStatsCompleter(CodaLabCompleter):
```

Comment:
Confused why there's a completer...these are for autocomplete on the CLI, but is irrelevant for reporting information back from docker since the user is not specifying them.

Reply:
Oh I didn't know that. I just found `DockerImagesCompleter` is performing something similar, so referred to this example.